src/cli.ts

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
```

Comment:
Is Line 1 telling the shell to execute this script using node?

Reply:
Yes, it lets the file be executed via `./cli.js` rather than having to do `node cli.js`